stores/dotenv/store.go

```diff
@@ -0,0 +1,152 @@
+package dotenv //import "go.mozilla.org/sops/stores/dotenv"
+
+import (
+	"bufio"
+	"bytes"
+	"encoding/base64"
+	"encoding/gob"
+	"fmt"
+	"go.mozilla.org/sops"
+	"go.mozilla.org/sops/stores"
+	"strings"
+)
+
+const SopsPrefix = "sops_"
+
+// Store handles storage of dotenv data
+type Store struct {
+}
+
+func (store *Store) LoadEncryptedFile(in []byte) (sops.Tree, error) {
+	branch, err := store.LoadPlainFile(in)
+	if err != nil { return sops.Tree{}, err }
```

Comment:
Have you ran `gofmt` (or even better, `goimports`) on this file? Not sure if it forces it, but generally Go programmers put the if body in a new line, like so:

```go
if err != nil {
    return sops.Tree{}, err
}
```

Reply:
I'll run `goimports` on it, thanks for the pointer.